### DISCUSSION TITLE
HMR not working with VueDraggable

### DISCUSSION THREAD

**Discussion by barisener** [quasarframework/quasar#17086] [General - Components / Directives / etc]
when i use vue draggable library, hmr not working. (https://github.com/SortableJS/vue.draggable.next repo url)
I create a new project with quasar cli and just added vue draggable then hmr was broken. 
has anyone got this problem?

**Comment by jclaveau** [comment]
You may begin by using a maintained fork: https://github.com/Alfred-Skyblue/vue-draggable-plus

Does it work with it?

**Reply by barisener** [reply]
I updated Quasar CLI and it fixed. 
[Upgrade guide ](https://quasar.dev/quasar-cli-vite/upgrade-guide#quasar-app-vite-v2-beta-)
